services/src/main/java/io/druid/cli/PullDependencies.java

```diff
@@ -65,7 +65,38 @@
 
   private static final Set<String> exclusions = Sets.newHashSet(
       "io.druid",
-      "com.metamx.druid"
+      "com.metamx.druid",
+
+      // These should be included by druid and not pulled down in extensions
```

Comment:
not sure how this will be kept up-to-date. does something go wrong if these are not excluded ?